lib/monitor/run.js

```diff
@@ -82,7 +83,17 @@ function run(options) {
     });
   }
 
-  child = spawn.apply(null, spawnArgs);
+  if (executable === 'node') {
```

Comment:
I'm also adding `cmd.args[0].indexOf('-') === -1 &&` to this line so that spawns if there's a node argument being passed.